app/src/main/kotlin/com/wire/android/ui/authentication/verificationcode/ResendCodeText.kt

```diff
@@ -30,23 +30,39 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.text.style.TextDecoration
 import com.wire.android.R
+import com.wire.android.ui.theme.wireColorScheme
 import com.wire.android.ui.theme.wireDimensions
 import com.wire.android.ui.theme.wireTypography
 
 @Composable
-fun ResendCodeText(onResendCodePressed: () -> Unit, clickEnabled: Boolean, modifier: Modifier = Modifier) {
+fun ResendCodeText(
```

Comment:
`ResendCodeText` is also used on registration flows: `CreateAccountCodeScreen` (old flow) and `CreateAccountVerificationCodeScreen` (new flow), shouldn't we also apply this logic there?